packages/type/scss/_default-type.scss

```diff
@@ -40,13 +40,17 @@
   }
 
   a {
-    @if meta.global-variable-exists('carbon--theme') and
-      map.has-key($carbon--theme, 'link-01')
-    {
-      color: map.get($carbon--theme, 'link-01');
-    } @else {
-      color: #0062fe;
-    }
+    // Is this still needed? To re-enable this behavior in v11 we will have to have
+    // the type package to depend on @carbon/theme so this can @use the theme() mixin
+    // to provide the value configured for `link-primary`.
+    //
+    //   @if meta.global-variable-exists('carbon--theme') and
+    //     map.has-key($carbon--theme, 'link-primary')
+    //   {
+    //     color: map.get($carbon--theme, 'link-primary');
+    //   } @else {
+    color: #0062fe;
+    //   }
```

Comment:
I think maybe our best bet for this would be:

```scss
color: var(--cds-link-primary, #0062fe);
```

That way it will respect the value if it exists but has a fallback

Reply:
@joshblack Oh man great idea, thanks for this suggestion!

Reply:
Updated